templates/registry/markdown/snippet.md.j2

```diff
@@ -4,7 +4,7 @@
 <!-- markdownlint-capture -->
 <!-- markdownlint-disable -->
 {% import 'attribute_table.j2' as at %}{% import 'metric_table.j2' as mt %}{% import 'event_macros.j2' as event %}{% import 'resource_macros.j2' as resource %}
-{% if group.type == "event" %}{{ event.header(group) }}{% elif group.type == "resource" %}{{ resource.header(group) }}{% elif group.type=="metric" %}{{ mt.generate(group) }}{% endif %}{{ at.generate(group.attributes, tag_filter, attribute_registry_base_url, group.lineage.attributes) }}
+{% if group.type == "event" %}{{ event.header(group) }}{% elif group.type == "resource" %}{{ resource.header(group) }}{% elif group.type=="metric" %}{{ mt.generate(group) }}{% endif %}{{ at.generate(group.attributes, tag_filter, attribute_registry_base_url, group.lineage.attributes) }}{% if group.type == "event" %}{{ event.body(group.body) }}{% endif %}
```

Comment:
Nit: Should we move to a structure like:
```
{% if group.type == "X"}x_snippet(group, ...){% elif group.type == "Y"}y_snippet(group,...){% else %} attribute_table(group,...){% endif %}
```

And then define local macros for `x_snippet`, `y_snippet` etc?